index.html

```diff
@@ -1,121 +1,174 @@
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>                        
+				</button>
+			</div>
+			<div class="collapse navbar-collapse" id="myNavbar">
+				<ul class="nav navbar-nav navbar-inverse">
+					<li><a href="#what-we-do" data-toggle="collapse" data-target=".nav-collapse.in">What We Do</a></li>
+					<li><a href="#the-system" data-toggle="collapse" data-target=".nav-collapse.in">The System</a></li>
+					<li><a href="#about-us">About Us</a></li>
+					<li><a href="#contact-us">Contact Us</a></li>
+				</ul>
+			</div>
+		</div>
+	</nav>	
+		
+	<!--MAIN-->
+	<div class="container-fluid">	
+	<div class="row-fluid">
+	
+		<div class="col-md-6 col-md-offset-3">
+		<div class="container-fluid">
+		<img class="img-in-text img-responsive" src="images/irisInAfrica.jpg" alt="Iris in Africa"></div>
+		<br>
+		
+		<!--WHAT WE DO-->
+		<!-- <a class="anchor" id="what-we-do"></a> -->
+		<section id="what-we-do">
+		<h2 style="padding-top: 70px; margin-top: -70px;">What We Do</h2>
+		<hr/>
+		<p>
+			Polorum is solving the problem of mass communication in rural Africa using the technology in people’s pockets – the mobile phone.
+			Where traditional infrastructure is lacking, mobile usage in Africa is on the rise. We see this as an opportunity to create a speedy, cost-effective channel through which 						locals, NGOs and public services can deliver the messages they need over isolated, spread out populations.
+			Over 40,000 phone numbers can be stored in our system, which sends out regular texts to all the mobile phones in a community. Run by locals for locals, our system is 						sustainable because its messages are trusted. 
+		</p>
+		<p>Polorum will enable communications that can: </p>
+		<dl>
+			<dt>Save Lives</dt>
+			<dd class="indent">In the event of an emergency or disaster, rapid communications spread awareness and help locals protect themselves. </dd>
+			<dt>Educate</dt> 
+			<dd class="indent">Useful messages to give families a hand, from healthy recipes to first aid. </dd>
+			<dt>Boost Local Economies</dt> 
+			<dd class="indent">Basic advertising services can help locals increase productivity and expand small businesses. </dd>
+		</dl>
+		</section>
+		<br>
+		
+		<!--THE SYSTEM-->
+		<!--<a class="anchor" id="the-system"></a>-->
+		<section id="the-system">
+		<h2 style="padding-top: 70px; margin-top: -70px;">The System</h2>
+		<hr/>
+		<p>
+  			<ul>
+  				<li> Is cheap and easy to use.</li>
+  				<li>Runs without internet or mains electricity.</li>
+  				<li>Fits into pre-existing infrastructure.</li>
+  				<li>Protects data by design.</li>
+  			</ul>
+			Need support? We can provide the technical expertise and training to get the system up and running in a community so information can be transmitted in seconds, rather than months.
+		</p>
+		<img class="img-in-text img-responsive" src="images/Oonagh_Helen_props.JPG" alt="Oonagh & Helen">
```

Comment:
Yup. I think you're right. I just got scared by github's higlighting it red, all good